!!! init_tp by default: 
!!! Using z3
Starting z3... 

Last Proving Location: 1 File "ho/wait2z.slk",Line:7,Col:10

ERROR: at ho/wait2z.slk_7:10_7:32
Message: compute_view_fw_bw: data not found: WAIT

!!! WARNING logtime exception:0.
Entail 1: EXCast. Globals.Illegal_Prover_Format("z3.smt_of_exp: ERROR in constraints (set/tup2) should not appear here :  {tup2_1_2,tup2_1_2}")

!!! WARNING logtime exception:0.
Entail 2: EXCast. Globals.Illegal_Prover_Format("z3.smt_of_exp: ERROR in constraints (set/tup2) should not appear here :  {tup2_1_2,tup2_2_1}")

!!! WARNING logtime exception:0.
Entail 3: EXCast. Globals.Illegal_Prover_Format("z3.smt_of_exp: ERROR in constraints (set/tup2) should not appear here :  {tup2_1_2,tup2_2_1}")

!!! WARNING logtime exception:0.004
Entail 4: EXCast. Globals.Illegal_Prover_Format("z3.smt_of_exp: ERROR in constraints (set/tup2) should not appear here :  {tup2_1_2,tup2_1_2}")

!!! WARNING logtime exception:0.
Entail 5: EXCast. Globals.Illegal_Prover_Format("z3.smt_of_exp: ERROR in constraints (set/tup2) should not appear here :  {tup2_1_2,tup2_1_2}")

